.github/workflow/deploy.yml

```
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - main
+
+name: 배포
```

```suggestion
name: 배포

on:
  push:
    branches:
      - main
```

이렇게 하는 게 더 읽기 좋아 보여요

---

첫 GitHub actions 작성 아주 굿굿